highfidelity/hifi
# Make the Appreciate App an uncertified default script

# Test Plan
I don't know who will handle QA testing on this PR, since it is a Platform team PR that came from Experiences. I'll have to figure that out.

Here's the test plan:
1. _Cleanly_ install Interface from a build of this PR. Launch Interface.
    1. Verify that the Appreciate app is running.
2. Perform a quick smoke test of the Appreciate app.
    1. Verify that the Appreciate app works as expected.
3. From Running Scripts, stop `appreciate_app.js`.
    1. Verify that the app has stopped.
4. Close and restart Interface.
    1. Verify that the Appreciate app is **not running.**

# More Details
We want a released, uncertified version of the Appreciate App to run by default for new users.

We also want to copy the entirety of the Appreciate app source code into the `hifi` repo so that it will be copied to a user's local disk when they install Interface.

If a user installs a version of Appreciate from the Marketplace (or from anywhere besides the default scripts location), the baked-in version of Appreciate will stop running.
